src/config.c

```diff
@@ -3114,6 +3114,7 @@ standardConfig static_configs[] = {
     createBoolConfig("aof-disable-auto-gc", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, server.aof_disable_auto_gc, 0, NULL, updateAofAutoGCEnabled),
     createBoolConfig("replica-ignore-disk-write-errors", NULL, MODIFIABLE_CONFIG, server.repl_ignore_disk_write_error, 0, NULL, NULL),
     createBoolConfig("hide-user-data-from-log", NULL, MODIFIABLE_CONFIG, server.hide_user_data_from_log, 0, NULL, NULL),
+    createBoolConfig("avoid-arbitrary-lazyexpires", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, server.avoid_arbitrary_lazyexpires, 0, NULL, NULL),
```

Comment:
i think we should do one of the following:
1. call it `lazyexpire-nested-arbitrary-keys`
2. call it `lazyexpire-arbitrary-keys` and it will get `yes, no, only-direct`

Reply:
i vote for 1